docs/developers/protocol/hermez-protocol/fee-table.md

```diff
@@ -4,532 +4,537 @@
 
 Fees in L2 transaction apply a factor encoded by an index, as shown in the table above and the formula below:
 
-$0 \text{ if } i = 0$
+$0 \quad \text{ if } i = 0$
 
-$10^{-24 + \frac{i}{2}} \text{if } 1 \leq i \leq 32$
+$2^{-60 + i·(\frac{-8-(-60)}{32})} \quad \text{if} \quad 1 \leq i \leq 31$
 
-$10^{-8 + \frac{8}{192} (i - 32) } \text{if } 33 \leq i \leq 223$
+$2^{-8 + (i-32)·(\frac{0-(-8)}{160})} \quad \text{if} \quad 32 \leq i \leq 191$
```

Comment:
It's a bit strange to express the fraction like that, instead of its simplified form.